### DISCUSSION TITLE
How to choose style in Dreambooth?

### DISCUSSION THREAD

**Discussion by surelymz** [TheLastBen/fast-stable-diffusion#2305] [Q&A]
I saw the style box here when I rendered the UI. However, when I click on the drop box, it shows no options. If I want to generate anime-style images, what should I do? 

![image](https://github.com/TheLastBen/fast-stable-diffusion/assets/114205829/4ea62e9b-3e2c-4977-b8d3-3884071a1a3d)

Many thanks in advance!

**Comment by i-huzaifa-arshad** [comment]
I think you have to save your prompts and settings by clicking the floppy disk icon, and then it will be available in styles.
